Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -259,7 +259,7 @@
                     .withFixedValueRange(createActiveRange(), false)
                     .withRendering(dataItem => Helpers.RenderFieldsHelper.renderActiveNotActive(dataItem.ActiveAccordingToValidityPeriod))
                     .withContentAlignment(Utility.KendoGrid.KendoColumnAlignment.Center)
-                    .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.frontPage)))
+                    .withInclusionCriterion(() => uiState.isBluePrintNodeAvailable(uiBluePrint.children.frontPage.children.dateFields)))
```

Comment:
what about the individual date fields... would like their columns to be hidden as well

Reply:
II could only find a column for "concluded" (ibrugtagningsdato) but that should be covered by this condition as well

Reply:
it's covered 
![image](https://user-images.githubusercontent.com/101557040/191898947-96c2154d-7343-4bb1-a4dc-8d295e63ead6.png)

Reply:
@a-naskret Excellent